### ISSUE TITLE
AlchemyAPI Deprecated, Watson NLU is now the preferred way

### ISSUE THREAD

**Issue by miltontony** [Code4SA/mma-dexter#59]
I've run into an issue use the Alchemy extractor. It requires an API key which is no longer provided as AlchemyAPI has been deprecated.

Are there any plans to switch to Wastson NLU?

**Comment by mattadendorff** [comment]
Absolutely. I'm going to update Dexter to work with Watson early in the new year.

**Comment by miltontony** [comment]
Thanks. Are there any workarounds to running it without using the Alchemy extractor? Is there an easy way (or a surgical way) to disable it in the meantime?

**Comment by mattadendorff** [comment]
You can remove the AlchemyExtractor from the extractor list in the DocumentProcessor class (in the document_processor.py file) to disable calls to it. This will impact some information that goes into the DB, however, which may have consequences downstream.
